bamboo/unit_tests/test_unit_checkpoint.py

```diff
@@ -1,31 +1,40 @@
-import os, pytest, sys
+import sys
+sys.path.insert(0, '../common_python')
+import tools
+import pytest
+import os, sys
 
 def test_unit_checkpoint_lenet(cluster, exe, dirname):
-    slurm_cmd = 'srun -n1'
-    if os.getenv('SLURM_NNODES') is None:
-        slurm_cmd = 'salloc -N1 ' + slurm_cmd
-
-    if cluster in ['catalyst', 'surface']:
-        return_code_nockpt = os.system('%s %s --model=%s/model_zoo/tests/model_lenet_mnist_ckpt.prototext --optimizer=%s/model_zoo/optimizers/opt_sgd.prototext --reader=%s/model_zoo/data_readers/data_reader_mnist.prototext --num_epochs=2' % (slurm_cmd, exe, dirname, dirname, dirname))
-    else:
-        raise Exception("Unsupported Cluster %s" % cluster)
+    command = tools.get_command(
+        cluster=cluster, executable=exe, num_nodes=1, num_processes=1,
+        dir_name=dirname,
+        data_filedir_ray='/p/gscratchr/brainusr/datasets/MNIST',
+        data_reader_name='mnist', model_folder='tests',
+        model_name='lenet_mnist_ckpt', num_epochs=2, optimizer_name='sgd')
```

Comment:
Think you accidentally removed the following line here:

"return_code_nockpt = os.system(command)"

Reply:
Fixed.